keylime/mba/elchecking/elchecker.py

```diff
@@ -92,7 +92,7 @@ def bootlog_evaluate(
     if len(missing_pcrs) > 0:
         logger.error("PCRs specified for policy %s not in quote: %s", mb_policy_name, str(missing_pcrs))
         failure.add_event("missing_pcrs", {"context": "PCRs are missing in quote", "data": missing_pcrs}, True)
-
+        reason = None
```

Comment:
Good catch. Maybe I suggest to move it just before the `if`? If in the future this piece get refactored, would be easy to stop that both branches define the variable